contracts/src/v0.8/functions/dev/1_0_0/FunctionsBilling.sol

```diff
@@ -167,7 +167,7 @@ abstract contract FunctionsBilling is Routable, IFunctionsBilling {
 
     uint256 juelsPerGas = _getJuelsPerGas(gasPriceWithOverestimation);
     uint256 estimatedGasReimbursement = juelsPerGas * executionGas;
-    uint96 fees = donFee + adminFee;
+    uint96 fees = uint96(donFee) + uint96(adminFee);
```

Comment:
I checked that also casting just the first one works correctly (not sure if there's a generally preferred way though).